controls/3.7/def.cf

```diff
@@ -236,8 +236,9 @@ bundle common def
       # Internal CFEngine log files rotation
       "cfengine_internal_rotate_logs" expression => "any";
 
-      # Enable agent email output (also see mailto, mailfrom and smtpserver)
+      # Enable or disable agent email output (also see mailto, mailfrom and smtpserver)
       "cfengine_internal_agent_email" expression => "any";
+      "cfengine_internal_disable_agent_email" expression => "!any";
```

Comment:
I actually prefer to have both a positive and a negative class to control things. 

We should keep it at least for people with existing def.cf. If we really want to remove it then we can add some reports based on the usage of the class to warn people that it will be deprecated.